Kickstarter-iOS/DataSources/CategorySelectionDataSource.swift

```diff
@@ -8,12 +8,16 @@ internal final class CategorySelectionDataSource: ValueCellDataSource {
 
   weak var collectionView: UICollectionView?
 
-  func load(_ sectionTitles: [String], categories: [[String]]) {
+  func load(_ sectionTitles: [String], categories: [[(String, Int)]]) {
```

Comment:
Consider a typealias for (name: String, id: Int) so we can write value.name and value.id below